tests/integration/telemetry/policy/envoy_ratelimit_test.go

```diff
@@ -157,6 +181,31 @@ func setupEnvoyFilter(ctx resource.Context) error {
 	return nil
 }
 
+func cleanupEnvoyFilter(ctx resource.Context, isLocal bool) error {
+	file := "testdata/enable_envoy_ratelimit.yaml"
```

Comment:
Can we cache the generated content instead of generating it again? Or at least avoid code duplication?

Reply:
Good point.. fixed!